packages/babel-preset-react-app/index.js

```diff
@@ -92,16 +121,4 @@ if (env === 'test') {
     ],
     plugins: plugins
   };
-
-  if (env === 'production') {
-    // Optimization: hoist JSX that never changes out of render()
-    // Disabled because of issues:
-    // * https://github.com/facebookincubator/create-react-app/issues/525
-    // * https://phabricator.babeljs.io/search/query/pCNlnC2xzwzx/
-    // * https://github.com/babel/babel/issues/4516
-    // TODO: Enable again when these issues are resolved.
-    // plugins.push.apply(plugins, [
-    //   require.resolve('babel-plugin-transform-react-constant-elements')
-    // ]);
-  }
```

Comment:
I had to move the whole block to provide a function to `module.exports` in order to have access to `options` object